[2025-07-28T05:00:03.151+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T05:00:03.173+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_3123123123123.load_data_task manual__2025-07-28T04:59:59.392555+00:00 [queued]>
[2025-07-28T05:00:03.177+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_3123123123123.load_data_task manual__2025-07-28T04:59:59.392555+00:00 [queued]>
[2025-07-28T05:00:03.178+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-28T05:00:03.187+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data_task> on 2025-07-28 04:59:59.392555+00:00
[2025-07-28T05:00:03.194+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_3123123123123', 'load_data_task', 'manual__2025-07-28T04:59:59.392555+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag_generator_python.py', '--cfg-path', '/tmp/tmpbnanqqaf']
[2025-07-28T05:00:03.196+0000] {standard_task_runner.py:91} INFO - Job 15: Subtask load_data_task
[2025-07-28T05:00:03.309+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5804) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T05:00:03.310+0000] {standard_task_runner.py:63} INFO - Started process 5810 to run task
[2025-07-28T05:00:03.339+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_3123123123123.load_data_task manual__2025-07-28T04:59:59.392555+00:00 [running]> on host dfc351a2ad6c
[2025-07-28T05:00:03.396+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_3123123123123' AIRFLOW_CTX_TASK_ID='load_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T04:59:59.392555+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T04:59:59.392555+00:00'
[2025-07-28T05:00:03.397+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T05:00:03.404+0000] {logging_mixin.py:188} INFO - Loading data from Redis key: ***_data:dag_3123123123123
[2025-07-28T05:00:03.423+0000] {logging_mixin.py:188} INFO - Saved output CSV to: /opt/***/data/output/3f2a4b93-b237-4545-bd50-5004d863567c.csv
[2025-07-28T05:00:03.423+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-28T05:00:03.424+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T05:00:03.430+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_3123123123123, task_id=load_data_task, run_id=manual__2025-07-28T04:59:59.392555+00:00, execution_date=20250728T045959, start_date=20250728T050003, end_date=20250728T050003
[2025-07-28T05:00:03.442+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-28T05:00:03.452+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-28T05:00:03.455+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
